.travis.yml

```diff
@@ -1,15 +1,56 @@
+---
+sudo: required
+services:
+  - docker
 language: python
-
+python:
+  - "2.7"
+  - "3.4"
+env:
+  - OS_NAME=centos OS_VERSION=6
+  - OS_NAME=centos OS_VERSION=7
+  - OS_NAME=ubuntu OS_VERSION=trusty
+  - OS_NAME=ubuntu OS_VERSION=xenial
+before_install:
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt update
+  - sudo apt install -y docker-ce
+  - docker --version
 install:
-    - pip install tox
-
+  - wget http://archive.ubuntu.com/ubuntu/pool/universe/r/rpmlint/rpmlint_1.7-1_all.deb
+  - sudo dpkg -i rpmlint_1.7-1_all.deb || sudo apt install -f -y
+  - gem install mdl
```

Comment:
you're probably not going to have rubygems on the build environment. Maybe use https://pypi.org/project/pymarkdownlint/ ?

Reply:
It's working so i presume it is present :)
But it's an option, I will check

Reply:
No more develop: https://github.com/jorisroovers/pymarkdownlint